source/guide/3.2/ui-blueprint-upload.md

```diff
@@ -27,6 +29,11 @@ tar czf blueprint-name.tar.gz blueprint-folder/
 
 The output file of the tar command above will be `blueprint-name.tar.gz`.
 
+Supported file extension are:
```

Comment:
If we support multiple file formats, we need to remove "TAR" from everywhere else..